jobs/config.json

```diff
@@ -4667,6 +4645,26 @@
       "sig-network"
     ]
   },
+  "ci-kubernetes-e2e-gci-gce-ingress-ipvs": {
+    "args": [
+      "--check-leaked-resources",
+      "--cluster=",
+      "--env=KUBE_PROXY_MODE=ipvs",
```

Comment:
I think you still need an `--extract` flag

Reply:
Added it